SETUP_ORACLE.md

```diff
@@ -0,0 +1,39 @@
+# Setup Oracle
+
+## Install Oracle Instant Client
+
+1. Go to [the official Oracle Instant Client Downloads site](https://www.oracle.com/database/technologies/instant-client/downloads.html) and download the following **.rpm** packages for your operative system:
+
+  - Basic
+  - SQL Plus
+  - SDK
+
+2. Create a folder in `/opt/oracle` if it does not exist yet (with `mkdir -p /opt/oracle`) and save these packages there.
+
+3. Install the dependency `libaio1` and the package `alien`.
+In Ubuntu that is done with `sudo apt-get install libaio1 alien`.
+
+4. Go to `/opt/oracle` through the terminal (with `cd /opt/oracle`) and use alient to convert all those **.rpm** packages to **.deb** (with `sudo alien --to-deb --scripts *.rpm`).
```

Comment:
```suggestion
4. Go to `/opt/oracle` through the terminal (with `cd /opt/oracle`) and use **alien** to convert all those **.rpm** packages to **.deb** (with `sudo alien --to-deb --scripts *.rpm`).
```

Reply:
You can also download the zip files. Unzip them in a directory and point `ORACLE_HOME` environment to this directory
That's what we do on the Dockerfiles

Reply:
Cool, thanks! :muscle: